### PR TITLE
Refactor names and fixed few small issues

### PR DESCRIPTION
**Pull request by AndriyPt**

### CODE REVIEW

File: ansible/docker_site.yml [hunk 1]
```diff
@@ -12,11 +12,14 @@
...
     - {role: ci/init/check_cache, when: run_build, tags: ["check_cache"]}
-    - {role: ci/init/install, when: run_install, tags: ["install"]}
-    - {role: ci/init/create_workspace, when: run_install, tags: ["create_workspace"]}
-    - {role: ci/update_dependencies, when: run_build, tags: ["update_dependencies", "build", "unit_tests",
-                                                             "code_coverage", "codecov_tool", "python_code_coverage",
-                                                             "cpp_code_coverage", "check_deb", "check_build"]}
+    - {role: ci/init/install, when: run_install, tags: ["build", "unit_tests", "code_coverage", "codecov_tool", "python_code_coverage",
+                                                        "cpp_code_coverage", "check_deb", "check_build", "python_code_style_check",
+                                                        "cpp_code_style_check", "code_style_check"]}
+    - {role: ci/init/create_workspace, when: run_install, tags: ["build", "unit_tests", "code_coverage", "codecov_tool", "python_code_coverage",
```
Comment (ugocupcic) [inline ansible/docker_site.yml]: why is it the same as above?

File: ansible/roles/ci/common/packages/files/packages_list.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export has_packages=0
+
+cd $directory
+
+for file in $(find -type f -name 'package.xml');
+do
+    if [ $has_packages -eq 1 ]; then
+        export result="$result,"
+    fi
+    export package_name=$(grep -e '<name>' $file | sed -e 's,.*<name>\([^<]*\)</name>.*,\1,g')
```
Comment (ugocupcic) [inline ansible/roles/ci/common/packages/files/packages_list.sh]: can you put a comment above that line to explain what the `sed` voodoo does?

File: ansible/roles/ci/update_dependencies/tasks/main.yml [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ---
 # Playbook for workspace creation
 
+- name: Set dependencies_file
+  set_fact:
+    dependencies_file: "{{repo_sources_path}}/repository.rosinstall"
```
Comment (ugocupcic) [inline ansible/roles/ci/update_dependencies/tasks/main.yml]: and we're going to :cry: when merging that :)

File: ansible/docker_site.yml (hunk 1 above)
Comment (AndriyPt) [inline ansible/docker_site.yml]: roles install and create_workspace has the same dependencies. Basically they are needed for almoust any role. So I have removed their role tags and just added tags of all roles.

File: ansible/roles/ci/update_dependencies/tasks/main.yml (hunk 3 above)
Comment (AndriyPt) [inline ansible/roles/ci/update_dependencies/tasks/main.yml]: I'll need your help to fix all projects :)

File: ansible/docker_site.yml (hunk 1 above)
Comment (ugocupcic) [inline ansible/docker_site.yml]: but create_workspace doesn't depend on install?

Comment (AndriyPt) [inline ansible/docker_site.yml]: yep, but it doesn't have tag to invoke it separately. In docker_site.yml it is after install so this is some kind of dependency written not using more standard Ansible approach.

**Pull request merged**